pkgs/development/compilers/gcc/5/default.nix

```diff
@@ -547,4 +547,10 @@ stdenv.mkDerivation ({
 // optionalAttrs (!stripped || cross != null) { dontStrip = true; NIX_STRIP_DEBUG = 0; }
 
 // optionalAttrs (enableMultilib) { dontMoveLib64 = true; }
+
+# This is only here because it was here in previous commits and we
+# want to avoid a meaningless mass rebuild.  These variables get
+# overridden via crossAttrs in an actual cross-build.
+// { crossMingw = false; crossStageStatic = true; }
+
```

Comment:
I'm also in favor of mass rebuild here. Mass-rebuild patches are merged into staging, so they don't disturb normal much.

Also if we consider the fact that we'll remove that dead code in the future, that will cause mass rebuild anyway. Don't see any reason to wait.